assets/sass/common/shared/o-forms/_o-form.scss

```diff
@@ -444,7 +444,7 @@ only screen and (min-resolution: 2dppx) {
   .custom-checkbox {
     label {
       background: url('../img/ui/forms/checkbox-01@2x.png') no-repeat;
-      background-size: 50px 1055px;
+      background-size: 50px 1155px;
```

Comment:
Oh, hmmm, that's confusing - we should probably port that image back to courage as well (doesn't need to be now). Main concern is that at some point we're going to update assets/sass from courage, and we'll just overwrite this change and have a regression.

Can you move this css to one of the widget-specific sass files then? i.e. https://github.com/okta/okta-signin-widget/blob/master/assets/sass/modules/_forms.scss